include/deal.II/base/template_constraints.h

```diff
@@ -599,8 +599,6 @@ template <typename T> struct EnableIfScalar<std::complex<T> >
 
 
 
-
-
```

Comment:
Nah, I think they're spoiled now. Like you said, they'll never be the same again now :-(